--- /rocker_scripts/init_userconf.sh	2022-09-27 14:52:45.000000000 +0100
+++ /rocker_scripts/init_userconf.sh	2022-09-27 14:52:53.000000000 +0100
@@ -64,7 +64,7 @@
     usermod -l "$USER" -d /home/"$USER" "$DEFAULT_USER"
     groupmod -n "$USER" "$DEFAULT_USER"
     usermod -a -G staff "$USER"
-    chown -R "$USER":"$USER" /home/"$USER"
+    # chown -R "$USER":"$USER" /home/"$USER"
     echo "USER is now $USER"
 fi
